pysmt/solvers/z3.py

```diff
@@ -146,7 +147,7 @@ def pop(self, levels=1):
     def print_model(self, name_filter=None):
```

Comment:
Unrelated: It seems likne we have multiple implementations of print_model in each solver. We should factorize these.